### PR TITLE
feat: rename `vyper.interfaces` to `ethereum.ercs`

### PR DESCRIPTION
**Pull request by ControlCplusControlV**
### What I did

Fixed Issue #3675 

### How I did it

Honestly I just found and replaced, and it didn't break(??). Would double check but this seems to be working to me

### How to verify it

Run the tests, they seem to pass?

### Commit message

Aliased `vyper.interfaces` import to `ethereum.ercs`

### Description for the changelog

Aliased `vyper.interfaces` import to `ethereum.ercs`

### Cute Animal Picture

This is who you're bullying when your request changes on my PR (jk ... pls do whatever is necessary) 
image

### CODE REVIEW

File: docs/release-notes.rst [hunk 1]
```diff
@@ -927,7 +927,7 @@ Here is the old changelog:
 * **2019.04.05**: Add stricter checking of unbalanced return statements. (`#590 <https://github.com/vyperlang/vyper/issues/590>`_)
 * **2019.03.04**: ``create_with_code_of`` has been renamed to ``create_forwarder_to``. (`#1177 <https://github.com/vyperlang/vyper/issues/1177>`_)
 * **2019.02.14**: Assigning a persistent contract address can only be done using the ``bar_contact = ERC20(<address>)`` syntax.
-* **2019.02.12**: ERC20 interface has to be imported using ``from vyper.interfaces import ERC20`` to use.
```
Comment (charles-cooper) [inline docs/release-notes.rst:930]: i don't think we should change this changelog entry

Comment (ControlCplusControlV) [reply docs/release-notes.rst]: oops fixed

**Comment by charles-cooper** [comment]
this seems fine. do we want ERC20, ERC721, etc or IERC20, IERC721, etc? @fubuloubu

**Comment by ControlCplusControlV** [comment]
ERC20, ERC721, etc. Unfortunately it seems like the masses prefer it, and it has also grown on me, although do we risk namespace collisions with other libs if the stdlib is canonically taking up `ERC20`?

**Comment by fubuloubu** [comment]
> ERC20, ERC721, etc. Unfortunately it seems like the masses prefer it, and it has also grown on me, although do we risk namespace collisions with other libs if the stdlib is canonically taking up `ERC20`?

It's namespaced behind a module, so can always rename the import

**Comment by fubuloubu** [comment]
> this seems fine. do we want ERC20, ERC721, etc or IERC20, IERC721, etc? @fubuloubu

As is e.g. `ERC20`

**Review by fubuloubu — APPROVED** [review]

**Review by charles-cooper — APPROVED** [review]
thanks!

**Pull request merged**